tests/nested/core20/gadget-reseal/task.yaml

```diff
@@ -0,0 +1,50 @@
+summary: Check that a gadget refresh reseals
+
+systems: [ubuntu-20.04-64]
+
+execute: |
+    # Wait for snapd to be seeded
+    nested_exec sudo snap wait system seed.loaded
```

Comment:
This line should be removed, otherwise there's this error:
```
2020-09-16 10:20:48 Error executing google-nested:ubuntu-20.04-64:tests/nested/core20/gadget-reseal :                                                                                                                                                                                                                                                                                          
-----                                                                                                                                                                                                                                                                                                                                                                                          
+ nested_exec sudo snap wait system seed.loaded                                                                                                                                                                                                                                                                                                                                                
/bin/bash: line 76: nested_exec: command not found  
```